cartographer_ros/cartographer_ros/color.cc

```diff
@@ -14,20 +14,21 @@
  * limitations under the License.
  */
 
-#include "cartographer_ros/color_manager.h"
+#include "cartographer_ros/color.h"
 
 #include <cmath>
 
 #include "glog/logging.h"
 
-namespace cartographer_ros {
+namespace {
```

Comment:
No need to move this out of the `cartographer_ros` namespace, see e.g. `msg_conversion.cc`.